ql/src/java/org/apache/hadoop/hive/ql/exec/FileSinkOperator.java

```diff
@@ -1617,8 +1617,7 @@ public OperatorType getType() {
 
   @Override
   public void augmentPlan() {
-    PlanUtils.configureOutputJobPropertiesForStorageHandler(
-        getConf().getTableInfo());
+    PlanUtils.configureOutputJobPropertiesForStorageHandler(getConf().getTableInfo());
```

Comment:
Refrain from formatting only changes, if not touching the file - this would just complicate backports

Reply:
Sure, reverted it